app.js

```diff
@@ -23,11 +23,9 @@ function removeDom(element) {
 
 function render(book) {
   awesomeBooks.innerHTML += `
-      <li id="${book.id}">
-        <h3>${book.title}</h3>
-        <h4>${book.author}</h4>
+      <li class="entry" id="${book.id}">
+        <p>"${book.title}" by ${book.author}</p>
         <button class="btn">Remove</button>
-        <hr>  
       </li>
```

Comment:
- [ ] As in this project you have to refactorize your functions to classes. Kindly add your functions inside a class and you can use your actual functions as a method of the classes, in this way you can have your project as OOP approach 🥇 

![image](https://user-images.githubusercontent.com/93630700/205951028-f8439531-d309-497c-9b1b-20678511ad6c.png)